packages/testing/src/steps/generic.steps.ts

```diff
@@ -161,6 +161,10 @@ export function setupGenericSteps() {
     expect(handleResolve(field, this)['message']).toBe(errorType);
   });
 
+  Then('{string} is an error', function (this: PropsWorld, field: string) {
+    expect(handleResolve(field, this)).toBeInstanceOf(Error);
```

Comment:
Why not use the existing 'error with message' and assert that the error matches "Invalid arguments passed to addContextListener!"?

Reply:
Because the error message is non-standardized. There doesn't seem any point in enforcing the arbitrary message. Instead I'd rather pursue this in 2.3 and then do it properly:
- #1490

Reply:
WIll add a comment to the test and code regarding that.

Reply:
done